source/switching_to_live/set_up_a_live_worldpay_account/index.html.md.erb

```diff
@@ -37,13 +37,13 @@ If __Test Mode__ is at the bottom of the left-hand menu, switch to __Production
 
 3. In your Worldpay account, select __Account__ and the __Profile__ tab, then your merchant code if you're asked for one.
 
-4. Copy the __Merchant Code__ value from the __Identification__ section of your Worldpay account into the __Merchant code__ field in your GOV.UK Pay account.
+4. In your Worldpay account, copy the __Merchant Code__ value from the __Identification__ section. Pastes this value into the __Merchant code__ field in your GOV.UK Pay account.
```

Comment:
Typo, "Pastes" -> "Paste"

Reply:
Whoops. Should not have done this in a hurry. Well-spotted, will change.

Reply:
Done @stephencdaly